### DISCUSSION TITLE
Swap mouse buttons

### DISCUSSION THREAD

**Discussion by knight1009** [rustdesk/rustdesk#910] [Feature Request]
Hi... Cool app... But I miss the option to swap the mouse button. Reason is that the remote-pc has a left-hand-mouse with swapped buttons... Not so easy to handle...

**Comment by rustdesk** [comment]
Thanks, we do need to think more about right-hand user.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/commit/2e146190e106df206ab89f02a3998431d8a715a5

**Comment by 21pages** [comment]
![1702270654136](https://github.com/rustdesk/rustdesk/assets/14891774/4f3baa5e-1445-4664-b446-ce3e71987038)

**Reply by 809825706** [reply]
how to use it now?

**Comment by roberto-chang-silva** [comment]
Hello, i was wondering if there is any update on this. For some reason unknown to me, what @21pages showed does not appear in my rustdesk menu.

**Reply by rustdesk** [reply]
nightly build or 1.2.4

**Reply by roberto-chang-silva** [reply]
Oh thank you!

**Reply by roberto-chang-silva** [reply]
I've tested! works perfect! thanks!
